### ISSUE TITLE
Iphone x Error -77

### ISSUE THREAD

**Issue by achavezl911** [checkra1n/BugTracker#1230]
Model: Iphone x
Version iOS: 13.4.1
Os: bootra1n
What version of checkra1n are you using? 0.10.1
Connected: Direct to pc 

Was working fine until today which uploaded from 13.2.3
Any other info ?

**Comment by wirasukma18** [comment]
The key is, remove all your passcode before jailbreak. Release all, and off it. Then, jailbreak again. Good luck.
![065A93F4-A1AB-494B-89FD-52B009279C61](https://user-images.githubusercontent.com/63896852/79630133-b8d9b700-8189-11ea-98e6-67056026017d.png)

**Comment by bineinkevin** [comment]
> The key is, remove all your passcode before jailbreak. Release all, and off it. Then, jailbreak again. Good luck.
> 
> ![065A93F4-A1AB-494B-89FD-52B009279C61](https://user-images.githubusercontent.com/63896852/79630133-b8d9b700-8189-11ea-98e6-67056026017d.png)
> 
> 

I can confirm that this method works. But it is in the bootra1n GitHub too

**Comment by wirasukma18** [comment]
> > The key is, remove all your passcode before jailbreak. Release all, and off it. Then, jailbreak again. Good luck.
> > ![065A93F4-A1AB-494B-89FD-52B009279C61](https://user-images.githubusercontent.com/63896852/79630133-b8d9b700-8189-11ea-98e6-67056026017d.png)
> 
> I can confirm that this method works. But it is in the bootra1n GitHub too

Yes, please use bootra1n linux live flashdisk. All works with fine, but, maybe bug, my iPad now overheating.

**Comment by syahrulgrey** [comment]
I allready remove my passcode but it keep eror 77
Why? 

Now im running on ios 13.4
